digits/templates/helper.html

```diff
@@ -51,3 +51,12 @@
         {{ ' has-warning' if form.warnings else '' }}
     {% endfor %}
 {% endmacro %}
+
+{% macro serve_file(obj, path) %}
+    {% if path[0] == '/' %}
+        {# Absolute path - don't serve file for security reasons #}
+        {{path}}
+    {% else %}
```

Comment:
Sorry for being picky... if the `.txt` file is in your DIGITS installation, say `$DIGITS_HOME/test/train.txt` and you specify the relative path to the file then a broken link will show on the dataset page (in addition to the fact that the job fails as the create DB script can't locate the relative text file). I don't think that is a major issue but should we not ban relative server paths?